rllib/env/multi_agent_env_runner.py

```diff
@@ -167,7 +169,7 @@ def sample(
                 (`num_episodes != None`), this is fixed to True.
 
         Returns:
-            A list of `MultiAgentEpisode` instances, carrying the sampled data.
+            A list of `SingleAgentEpisode` instances, carrying the sampled data.
```

Comment:
wait, this change is incorrect, no? It's still returning `List[MultiAgentEpisode]`.

Reply:
Correct. Great catch!